### PR TITLE
Implement Mailchimp backup

### PR DESCRIPTION
**Pull request by caldwellst**
Addresses #101.

### CODE REVIEW

**Review by hannahker — CHANGES_REQUESTED** [review]
Looks good! I would just suggest putting the script under `src/email/mailchimp`, since I think that directory is a good fit and we don't need to add any new directories.

**Comment by caldwellst** [comment]
Alright, moved under `email/mailchimp`!

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ documentation for details on datasets included and how signals are generated.
 
 The code is structured as below. Refer to the directories for specific details.
 
+- runs: Scripts for actual runs, called in `.github` workflows.
```
Comment (hannahker) [inline README.md:17]: Can remove this now!

**Review by hannahker — APPROVED** [review]
Looks great! Just see the comment to update the `README`, then should be good to merge.

File: README.md (hunk 1 above)
Comment (caldwellst) [reply README.md:17]: Doh! Good catch.

**Pull request merged**